src/firmware/main/io/chicker.c

```diff
@@ -473,13 +473,26 @@ static void chicker_fire_impl(chicker_device_t device, unsigned int width)
  * \param[in] device which device to fire
  * \param[in] width the width of the pulse, in kicking units
```

Comment:
lmao kicking units :eyes:

Reply:
:grimacing: - out of scope, but nasty.........